Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -356,32 +356,34 @@ public boolean isLimelightValid(Vision vision, double horizAngleRange, double di
   // Autonomous mode - Holonomic path follower
   //
 
-  public Trajectory PathPlannerTrajectorytoTrajectory(PathPlannerTrajectory trajectory)
+  public Trajectory PPTrajtoWPITraj(PathPlannerTrajectory trajectory)
```

Comment:
I realize I said to shorten the name, but "Traj" is probably a little too far. If it was a local variable (used within a method), then more cryptic names are OK. But a public method nameshould be more complete. PPTrajectoryToWPITrajectory is OK without the full PathPlanner name.